jumpscale/sals/jukebox/jukebox_deploy_chatflow.py

```diff
@@ -1,31 +1,48 @@
+from contextlib import ContextDecorator
 import random
 from textwrap import dedent
 
+from jumpscale.clients.explorer.models import DiskType
 from jumpscale.clients.stellar import TRANSACTION_FEES
 from jumpscale.loader import j
 from jumpscale.sals.chatflows.chatflows import GedisChatBot, StopChatFlow, chatflow_step
 from jumpscale.sals.marketplace.apps_chatflow import MarketPlaceAppsChatflow
 
-from jumpscale.packages.jukebox.sals import jukebox
+from jumpscale.sals.jukebox import utils
+from jumpscale.sals.jukebox.models import State
 
 IDENTITY_PREFIX = "jukebox"
 
 
+class new_jukebox_context(ContextDecorator):
+    def __init__(self, instance_name, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.instance_name = instance_name
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        if exc_type:
+            j.logger.error(f"new_jukebox_context: deployment failed due to exception: {exc_value}")
+            j.sals.jukebox.delete(self.instance_name)
+
+
 class JukeboxDeployChatflow(MarketPlaceAppsChatflow):
     ENTRY_POINT = ""
+    DISK_TYPE = DiskType.SSD
     title = "Blockchain"
     steps = [
         "get_deployment_name",
         "block_chain_info",
         "choose_farm",
         "set_expiration",
-        "upload_public_key",
         "payment",
         "deploy",
         "success",
     ]
     # FLIST = "<flist_url>"
```

Comment:
```suggestion
```

Reply:
We need this for anyone adding a new child chatflow